src/encoded/static/components/GermlineTableNoState.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line linebreak-style
+import React from 'react';
+import { SortTablePanel, SortTable } from './sorttable';
+
+class GermlineTableNoState extends React.Component {
+    constructor(props) {
+        super(props);
+        // this.state = {
+        //     showData: true,
+        // };
+        //this.tableConfig = {};
+        this.data = this.props.data;
+        //this.zingchart = window.zingchart;
+        this.germlineFilters = [];
+        this.transferData = [];
+    }
+
+    render() {
+        if (!Array.isArray(this.data) || !this.data.length) {
+            return (
+                <div className="flex-container">
+                    <div className="chart-main">
+                        <div className="chart-title">
+                            <div><h3>{this.props.tableTitle}</h3> </div>
+                            <div><h4>Germline mutation data is not available!</h4></div>
+                        </div>
+                    </div>
+                </div>
+            );
+        } else {
+             this.germlineFilters = this.data.filter(i => (i.significance === 'Positive' || i.significance === 'Variant' || i.significance === 'Positive and Variant'));;
+            //this.transferData = this.germlineFilters.map(i => ({ values: [i.target, i.significance] }));
+            this.transferData = this.germlineFilters.map(i => ({ "target": i.target, "significance": i.significance}));

```

Comment:
transferData doesn't seem to be the right variable name. Maybe you meant transformedData?